linked audiences Iterable (#6769)
segmentio/segment-docs: src/engage/audiences/linked-audiences-iterable.md

|Segment Destination Action                                                                              |How does it work?                                                                                                                                                                                                                                                                                |How does Iterable store the data?                                                                                               |Iterable API Endpoint                                                 |
|--------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------|

> |[Custom Event](https://segment.com/docs/connections/destinations/catalog/actions-iterable/#custom-event){:target="_blank"}|Segment sends personalization payload information into Iterable as [Iterable custom events](https://support.iterable.com/hc/en-us/articles/206430615-Custom-Event-Properties#event-properties){:target="_blank"}. The entity personalization payload is contained in the `dataFields` parameter within API calls.|[Custom event properties](https://support.iterable.com/hc/en-us/articles/206430615-Events-and-Event-Properties#event-properties){:target="_blank"}|[Track an event](https://api.iterable.com/api/docs#events_track){:target="_blank"}      |

Reviewer: ```suggestion
|[Custom Event](/docs/connections/destinations/catalog/actions-iterable/#custom-event) |Segment sends personalization payload information into Iterable as [Iterable custom events](https://support.iterable.com/hc/en-us/articles/206430615-Custom-Event-Properties#event-properties){:target="_blank"}. The entity personalization payload is contained in the `dataFields` parameter within API calls.|[Custom event properties](https://support.iterable.com/hc/en-us/articles/206430615-Events-and-Event-Properties#event-properties){:target="_blank"}|[Track an event](https://api.iterable.com/api/docs#events_track){:target="_blank"}      |
```

## Iterable Journey Campaign

Iterable [Journey Campaigns](https://support.iterable.com/hc/en-us/articles/360050203812-Campaigns-Overview#journey-campaigns){:target="_blank"} store the entity personalization payload as [event properties in custom events](https://support.iterable.com/hc/en-us/articles/206430615-Events-and-Event-Properties#event-properties){:target="_blank"}.

**Note**: Using custom events can impact your Iterable [billing](https://support.iterable.com/hc/articles/205480345#custom-event-usage-metrics){:target="_blank"}.

### Prerequisites

In Segment, ensure you have:

* Set up your [Linked Audience](/docs/engage/audiences/linked-audiences/). 
* A copy of your Linked Audience event name and Iterable user id.

> * Set up [Iterable as an actions destination](https://segment.com/docs/connections/destinations/catalog/actions-iterable/){:target="_blank"}. 

Reviewer: ```suggestion
* Set up [Iterable as an actions destination](/docs/connections/destinations/catalog/actions-iterable/). 
```

* Set up a [Custom Event destination action](/docs/connections/destinations/catalog/actions-iterable/#custom-event) and the relevant trigger using the [Linked Audiences](/docs/engage/audiences/linked-audiences/) workflow. 

In Iterable, ensure you have:

* Permissions to access the application, or access to someone who has permissions.

> * Created all Segment profiles as Iterables profiles using the [Upsert User destination action](/docs/connections/destinations/catalog/actions-iterable/#upsert-user){:target="_blank"} via [Connections](/docs/connections/) or [Reverse ETL](/docs/connections/reverse-etl/). This is a requirement for them to receive a campaign triggered by a custom event.

Reviewer: ```suggestion
* Created all Segment profiles as Iterables profiles using the [Upsert User destination action](/docs/connections/destinations/catalog/actions-iterable/#upsert-user) through [Connections](/docs/connections/) or [Reverse ETL](/docs/connections/reverse-etl/). This is a requirement for them to receive a campaign triggered by a custom event.
```

![A screenshot of the test event page](/docs/engage/images/send-test-event.png) 

If the event is sent successfully to Iterable, you will see a `“message”: “success”` response in Segment.
2. Open Iterable and check the [Campaign tab in Messaging Insights](https://support.iterable.com/hc/en-us/articles/360052814452-Messaging-Insights#viewing-metrics){:target="_blank"} (Insights > Messaging Insights > Campaigns > _[Your Unique Campaign])_ to confirm that Iterable has sent the message. 
3. If your email doesn’t look the way you want it to, adjust the [Handlebars syntax](https://support.iterable.com/hc/en-us/articles/205480365-Personalizing-Templates-with-Handlebars#referencing-user-profile-and-event-fields-with-handlebars){:target="_blank"} in Iterable, and send another test event. See the following [personalization examples](#handlebars-examples-to-use-in-iterable){:target="_blank"} for more specific details. 
4. When you’re finished testing your campaign, proceed to [Enable your Linked Audience](/docs/engage/audiences/linked-audiences/#step-4-enable-your-linked-audience).

> ## Handlebars Examples to use in Iterable

Reviewer: ```suggestion
## Handlebars examples to use in Iterable
```

Use the following examples as context and information to experiment with setting up your campaign in Iterable.

The following is an example of what your payload data might look like with nested payload properties:

```js
{
   "event": "abandoned_carts_linked_audience",
   "properties": {
      "first_name": "Andrew",
      "last_name": "Shopper",
      "shopping_cart__id": "123",
      "shopping_cart__products": [
         {
            "id": "324",
            "product_name": "Premium Tennis Shoes",
            "product_price": "$140",
            "__entity": "product"
         },
         {
             "id": "489",
             "product_name": "Premium Jacket",
             "product_price": "$200",
            "__entity": "product"
          }
       ],
       "__entity": "cart"
     }
 }
```

> The following helps translate you payload data into Handlebars syntax:  

Reviewer: ```suggestion
The following helps translate your payload data into Handlebars syntax:  
```

* To reference a specific event property:
    * Use the following Handelbars syntax: <code>{% raw %}{{event_property_name}}{% endraw %}</code>
    * An example of this property might look like:  <code>{% raw %}{{first_name}}{% endraw %}</code>
* To reference nested event properties within an Array:
    * Use the following Handlebars syntax:  <code>{% raw %}{{event_property_name.[#_that_represents_specific_entry_in_array].nested_event_property_name }}{% endraw %}</code>
    * An example of this property might look like:  <code>{% raw %}{{shopping_cart__products[0].product_name}}{% endraw %}</code>

You can read more on how to reference [event properties](https://support.iterable.com/hc/en-us/articles/205480365-Personalizing-Templates-with-Handlebars#referencing-user-profile-and-event-fields-with-handlebars){:target="_blank"} and [nested properties](https://support.iterable.com/hc/en-us/articles/360031118392-Handlebars-FAQ#how-do-i-reference-items-in-a-nested-object){:target="_blank"} in Handlebars. 

> ### Basic Email Example

Reviewer: ```suggestion
### Basic email example
```

Use the  Segment custom event payload schema to build an abandoned cart email campaign that includes specific information for the product in a customer’s shopping cart.

When an email is sent, it lists the specific product and its related price in your customer’s shopping cart. It might look like the following screenshot: 

![A screenshot of an email, with the name, item, and price personalized.](/docs/engage/images/linked-cart-simple.png)

This is an example of what your email using HTML and [Handlebars syntax](https://support.iterable.com/hc/en-us/articles/205480365-Personalizing-Templates-with-Handlebars){:target="_blank"} might look like in Iterable:

{% raw %} 

```html
Hi {{first_name}},
<br />
<br />
Did you forget to checkout?<br />
<br />
We noticed you added some items to your shopping cart including this item: <br />
<br />
<b>Product Name: </b> 
{{shopping_cart__products.[0].product_name}}
<br />
<b>Product Price: </b> 
{{shopping_cart__products.[0].product_price}} USD
<br />
<br />

Quick, now is your chance to own this item before it sells out!

```
{% endraw %} 

> ### Advanced Email Example

Reviewer: ```suggestion
### Advanced email example
```

> Use the Segment custom event payload schema to build an abandoned cart email campaign where you can use [#each](https://support.iterable.com/hc/en-us/articles/205480365-Personalizing-Templates-with-Handlebars#iterating-over-all-values-each){:target="_blank"} to run a block of code repeatedly. In this example, you can list all of the products and their related prices in a customer’s shopping cart.

Reviewer: ```suggestion
Use the Segment custom event payload schema to build an abandoned cart email campaign where you can use the [`#each` block helper](https://support.iterable.com/hc/en-us/articles/205480365-Personalizing-Templates-with-Handlebars#iterating-over-all-values-each){:target="_blank"} to run a block of code repeatedly. In this example, you can list all of the products and their related prices in a customer’s shopping cart.
```
